src/g_game.c

```diff
@@ -368,22 +368,6 @@ static void G_DemoSkipTics(void)
   }
 }
 
-static double PitchToShearingHeight(double pitch)
```

Comment:
I need to work through the math again, but I believe some form of the angle-to-shearing-height conversion here is still needed otherwise horizontal look won't be 1:1 with vertical look.